queries.sql

```diff
@@ -38,4 +38,81 @@ WHERE name != 'Gabumon'
 /*  Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg) */
 
 SELECT * FROM animals
-WHERE weight_kg BETWEEN 10.4 AND 17.3 OR weight_kg = 10.4 AND weight_kg = 17.3;
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 AND 17.3 OR weight_kg = 10.4 AND weight_kg = 17.3;
+
+-- DAY 2 --
+
+/* Inside a transaction update the animals table by setting 
+the species column to unspecified. Verify that change was 
+made. Then roll back the change and verify that species 
+columns went back to the state before tranasction. */
+
+BEGIN;
+	UPDATE animals
+	SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+/* Update the animals table by setting the species column 
+to digimon for all animals that have a name ending in mon. */
+
+	UPDATE animals
+	SET species = 'digimon'
+	WHERE name LIKE '%mon';
+/* Update the animals table by setting the species column 
+to pokemon for all animals that don't have species already set. */
+
+	UPDATE animals
+	SET species = 'pokemon'
+	WHERE species ISNULL;
+COMMIT;
```

Comment:
* Kindly complete this transaction as the requirement states that you `Verify that change was made` and `persists after commit`. [Check requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Commit%20the%20transaction,persists%20after%20commit.)